client/keys/show.go

```diff
@@ -199,14 +192,9 @@ func validateMultisigThreshold(k, nKeys int) error {
 	return nil
 }
 
-func getBechKeyOut(bechPrefix string) (bechKeyOutFn, error) {
-	switch bechPrefix {
-	case sdk.PrefixAccount:
+func getHexKeyOut(bechPrefix string) (hexKeyOutFn, error) {
+	if bechPrefix == "" {
 		return MkAccKeyOutput, nil
-	case sdk.PrefixValidator:
-		return MkValKeyOutput, nil
-	case sdk.PrefixConsensus:
-		return MkConsKeyOutput, nil
 	}
 
 	return nil, fmt.Errorf("invalid Bech32 prefix encoding provided: %s", bechPrefix)
```

Comment:
We might want to change the error message here to something like:
```suggestion
	return nil, fmt.Errorf("hex encoding doesn't have Bech32 prefix, yet provided: %s", bechPrefix)
```

Reply:
Nice one.
I'll address that in my `gov` PR

Reply:
Pushed 🙌